cypress/support/pages/editor.pageObject.js

```diff
@@ -0,0 +1,55 @@
+import PageObject from '../PageObject';
+
+class EditorPageObject extends PageObject {
+  url = 'editor';
+
+  get articleTitle() {
+    return cy.getByDataCy('Article Title');
+  }
+
+  get articleBody() {
+    return cy.getByDataCy('Write your article (in markdown)');
+  }
+
+  get articleDescription() {
+    return cy.getByDataCy(`What's this article about?`);
+  }
+
+  get articleTag() {
+    return cy.getByDataCy('Enter tags');
+  }
+
+  get publishArticleButton() {
+    return cy.get('.btn');
+  }
+
+  get updateArticleButton() {
+    return cy.get('.btn');
```

Comment:
Similarly, the selector for `updateArticleButton` is very generic ('.btn'). Ensure that this selector targets the correct button, or use a more specific selector if possible.